app/models/distribution.rb

```diff
@@ -53,8 +53,8 @@ class Distribution < ApplicationRecord
       .includes(:partner, :storage_location, :line_items)
   }
   scope :this_week, -> do
-    where("issued_at >= :start_date AND issued_at <= :end_date",
-          start_date: Time.zone.today, end_date: Time.zone.today.sunday)
+    where("issued_at > :start_date AND issued_at <= :end_date",
```

Comment:
Will this affect our navbar that shows the pickups remaining in the week to show all the pickups instead of the ones remaining?

Reply:
Looks like it - is this not desirable?

Reply:
I should have deleted this comment. The next one I made I realized my mistake.